api/swagger.yml

```diff
@@ -70,6 +70,13 @@ components:
       description: delimiter used to group common prefixes by
       schema:
         type: string
+
+    PaginationSearchString:
+      in: query
+      name: search
```

Comment:
I think the name `search` is too vague.
Maybe `query` or `search-query` is more accurate in this context.

Moreover, I find it weird that a `list` API is getting some searching capabilities as a main part of its signature.
Maybe a structure containing some search params might be more clear, but it's tricky when using a GET request.

How about the following solution:
Supporting `*<search-term>*` pattern for the `prefix` param.
When the API encounters such pattern, it performs a substring search, and it's not really breaking the API.
And the WebUI can wrap the `prefix` with `*`s, and actually make such search.

Reply:
I got rid of PaginationSearchTerm- I agree that is just too much

Reply:
> I got rid of PaginationSearchTerm- I agree that is just too much

Yeah, it's not part of pagination.